ladybug/analysisperiod.py

```diff
@@ -271,6 +354,45 @@ def is_possible_hour(self, hour):
             return self.st_time.hour <= hour <= 23 or \
                 0 <= hour <= self.end_time.hour
 
+    def is_time_included(self, time):
+        """Check if time is included in analysis period.
+
+        Return True if time is inside this analysis period,
+        otherwise return False
+
+        Args:
+            time: A DateTime to be tested
+
+        Returns:
+            A boolean. True if time is included in analysis period
+        """
+        if self._timestamps_data is None:
+            self._calculate_timestamps()
+        # time filtering in Ladybug and honeybee is slightly different since
```

Comment:
That is a comment that you originally wrote and I had left unchanged since I didn't do anything that would require a change.  However, realizing how you just interpreted your own comment, I think that, in hindsight, you probably meant to say:
`# time filtering in Ladybug Tools is slightly different than "normal" filtering since`
I have edited the comment accordingly.